README

```diff
@@ -2627,7 +2628,29 @@ once or readded in the ScummVM launcher's game list. This will update the
 configuration of each entry, allowing the custom options to be shown.
 
 
-9.0) Compiling:
+9.0) Screenshots (SDL backend only):
+---- -------------------------------
+By default screenshots are put into the current directory, however on Windows
+it has a preset directory:
+
+    Windows Vista/7:
+    \Users\username\AppData\Roaming\ScummVM\Screenshots\
+
+    Windows 2000/XP:
+    \Documents and Settings\username\Application Data\ScummVM\Screenshots\
+
+    Windows NT4:
+    <windir>\Profiles\username\Application Data\ScummVM\Screenshots\
+    
+    Windows 95/98/ME:
+    <windir>\
+
+Screenshots are stored under a hidden area in Windows NT4/2000/XP/Vista/7,
```

Comment:
also keep in mind that you can just use SDL_GetPrefPath if sdl > 2.0 is used (and it looks like you are talking about the sdl backend here anyway).